### PR TITLE
Auto-scaling group tags and instance EBS size

### PR DESCRIPTION
**Pull request by cmilam87**

### CODE REVIEW

**Comment by andreaswittig** [comment]
Hi @cmilam87, thanks for your PR. Great idea, could you please the indentation?

**Comment by cmilam87** [comment]
Look good to you?

File: template.json [hunk 1]
```diff
@@ -425,6 +432,12 @@
         "KeyName": {"Fn::If": ["HasKeyNameParameter", {"Ref": "KeyNameParameter"}, {"Ref": "AWS::NoValue"}]},
         "AssociatePublicIpAddress": true,
         "EbsOptimized": false,
+        "BlockDeviceMappings": [{
+	    	"DeviceName": "/dev/sda1",
+	    	"Ebs": {
+    			"VolumeSize": 8
+	    	}
+    	}],
```
Comment (Eugst) [inline template.json]: will it work for t2 instances? I had experience that `BlockDeviceMappings` make a problems for starting new type instances.

Comment (cmilam87) [inline template.json]: BlockDeviceMappings will work for any instance type.

**Pull request merged**